### PR TITLE
Feature/spaces administrators

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java [hunk 1]
```diff
@@ -1411,6 +1407,22 @@ public boolean isSpaceMember(String spaceId, String userId) {
   }
 
   @Override
+  public boolean isSpaceSuperManager(String userId) {
```
Comment (thomasdelhomenie) [inline wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java]: It does not make a lot of sense to have a method isSpaceSuperManager on Wiki API (same as the method spaceMember). Why not calling Social API directly ?

Comment (boubaker) [inline wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java]: As you said the existing methods won't add a dependency to social artifacts. I will attempt to change this to see why it was done like that before.

Comment (boubaker) [inline wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java]: changed to improve old code as well

Comment (boubaker) [inline wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java]: done

File: wiki-service/src/main/java/org/exoplatform/wiki/service/WikiService.java [hunk 2]
```diff
@@ -605,6 +605,14 @@
   public boolean isSpaceMember(String spaceId, String userId);
 
   /**
+   * Checks if a given user is super manager of space or not.
+   *
+   * @param userId The username.
+   * @return "True" if the user is super manager, or "false" if not.
+   */
+  public boolean isSpaceSuperManager(String userId);
```
Comment (thomasdelhomenie) [inline wiki-service/src/main/java/org/exoplatform/wiki/service/WikiService.java]: To be removed.

Comment (boubaker) [inline wiki-service/src/main/java/org/exoplatform/wiki/service/WikiService.java]: done

**Pull request merged**